vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -1064,9 +1064,16 @@ public void deletePictureResource(String uri, ModelCallback callback) {
 
     @Nullable
     public retrofit2.Response<Video> fetchVideoSync(String uri, @Nullable String fieldFilter) {
+        return fetchVideoSync(uri, CacheControl.FORCE_NETWORK, fieldFilter);
+    }
+
+    @Nullable
+    public retrofit2.Response<Video> fetchVideoSync(String uri, CacheControl cacheControl,
```

Comment:
Would you want to add some JavaDocs to this method/one above (so we can one day strive for total javadocing of our open source libs)? 🌞